### ISSUE TITLE
 make error

### ISSUE THREAD

**Issue by brian0603** [SimpleSSD/SimpleSSD-Standalone#5]
Hi,
i follow the instruction in the website.
There are no problems with the previous steps.
But,when i input "make -j 8",it occurs the following errors.
would you help me to solve this problem ?  thanks

**[100%] Linking CXX executable simplessd-standalone
CMakeFiles/simplessd-standalone.dir/sim/main.cc.o: In function `std::thread::thread<void (&)(int), int&>(void (&)(int), int&)':
main.cc:(.text._ZNSt6threadC2IRFviEJRiEEEOT_DpOT0_[_ZNSt6threadC5IRFviEJRiEEEOT_DpOT0_]+0x63): undefined reference to `pthread_create'
CMakeFiles/simplessd-standalone.dir/sim/signal.cc.o: In function `raiseSignal(int)':
signal.cc:(.text+0xe): undefined reference to `pthread_kill'
libmcpat.a(Ucache.cc.o): In function `std::thread::thread<void (&)(void*), void*>(void (&)(void*), void*&&)':
Ucache.cc:(.text._ZNSt6threadC2IRFvPvEJS1_EEEOT_DpOT0_[_ZNSt6threadC5IRFvPvEJS1_EEEOT_DpOT0_]+0x63): undefined reference to `pthread_create'
collect2: error: ld returned 1 exit status
CMakeFiles/simplessd-standalone.dir/build.make:928: recipe for target 'simplessd-standalone' failed
make[2]: *** [simplessd-standalone] Error 1
CMakeFiles/Makefile2:68: recipe for target 'CMakeFiles/simplessd-standalone.dir/all' failed
make[1]: *** [CMakeFiles/simplessd-standalone.dir/all] Error 2
Makefile:83: recipe for target 'all' failed
make: *** [all] Error 2**

**Comment by kukdh1** [comment]
Hi,

Please try to change `CMakeLists.txt` as following:
```diff
@@ -56,14 +56,12 @@ if (MSVC)
 else ()
   if (DEBUG_BUILD)
     set(CMAKE_CXX_FLAGS
-      "-O0 -g -rdynamic -Wall -Wextra -Werror ${CMAKE_CXX_FLAGS}")
+      "-O0 -g -rdynamic -pthread -Wall -Wextra -Werror ${CMAKE_CXX_FLAGS}")
   else ()
     set(CMAKE_CXX_FLAGS
-      "-O2 -rdynamic -Wall -Wextra -Werror ${CMAKE_CXX_FLAGS}")
+      "-O2 -rdynamic -pthread -Wall -Wextra -Werror ${CMAKE_CXX_FLAGS}")
   endif ()
 
-  set(CMAKE_EXE_LINKER_FLAGS "-lpthread ${CMAKE_EXE_LINKER_FLAGS}")
-
   if (CMAKE_CXX_COMPILER_ID STREQUAL "GNU")
     if (CMAKE_CXX_COMPILER_VERSION VERSION_LESS "4.9")
       message(FATAL_ERROR "GNU g++ < 4.9 does not support <regex>.\
```

Thanks

**Comment by brian0603** [comment]
it works. Thank you!!

**Comment by kukdh1** [comment]
Fixed in internal repo (d7e9b8ac).
Will be appear to public at tag `v2.0-rc4`.
